### PR TITLE
Fix ci for atomic scan

### PR DESCRIPTION
**Pull request by rtnpro**

### CODE REVIEW

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

File: provisions/roles/jenkins/master/tasks/jobs.yml [hunk 1]
```diff
@@ -20,13 +20,13 @@
 - name: Push job to jenkins
   shell: jenkins-jobs --ignore-cache update {{ ansible_env.HOME}}/index.yml
 
-- name: Ensure /opt/container-pipeline-service/jenkinsbuilder dir exists on Jenkins master
-  file: dest=/opt/container-pipeline-service/jenkinsbuilder state=directory
+- name: Ensure /opt/cccp-service/jenkinsbuilder dir exists on Jenkins master
+  file: dest=/opt/cccp-service/jenkinsbuilder state=directory
```
Comment (navidshaikh) [inline provisions/roles/jenkins/master/tasks/jobs.yml]: Why would we change the name of project?

Comment (rtnpro) [inline provisions/roles/jenkins/master/tasks/jobs.yml]: @navidshaikh please read the commit summary here: https://github.com/CentOS/container-pipeline-service/pull/58/commits/586a002d9dcb4759eb498201bfb875184e355b92

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
#dotests

**Comment by rtnpro** [comment]
Moved to https://github.com/CentOS/container-pipeline-service/pull/71

**Pull request closed without merging**